cryptography/src/commonJvmAndroid/kotlin/com.wire.kalium.cryptography/CoreCryptoCentral.kt

```diff
@@ -46,12 +46,12 @@ actual suspend fun coreCryptoCentral(
 
 private class Callbacks : CoreCryptoCallbacks {
 
-    override fun authorize(conversationId: ByteArray, clientId: ClientId): Boolean {
+    override suspend fun authorize(conversationId: ByteArray, clientId: ClientId): Boolean {
```

Comment:
Do we need it to be suspend  function ?

Reply:
yes those functions are not suspend from CC